#the header of the message
Header header
uint32[] id
sensor_msgs/PointCloud2 position
sensor_msgs/PointCloud2 velocity
sensor_msgs/PointCloud2[] obstacle_lines